src/routes/game/GameView.vue

```diff
@@ -472,7 +472,7 @@ export default {
     ...mapStores(useGameStore),
     ...mapStores(useAuthStore),
     isSpectating() {
-      return this.gameStore.isSpectating;
+      return this.gameStore.isSpectating || this.$router.currentRoute.value.name === ROUTE_NAME_SPECTATE;
```

Comment:
Do we want both the store value and the route or should we update the consumers of the store value to use the router instead?

Reply:
If the store value can be derived from the route it might make sense to keep both so you have one setter and all consumers use the store since that’s typically easier to check in the code.